server/utils/similarity.test.js

```diff
@@ -0,0 +1,21 @@
+const Similarity = require('./similarity')
+
+test('Test 0: propertly returns correct similarity method', () => {
+  expect(Similarity.getSimilarityMethod('62da14f88d6e09b9d06dace3')).toBe(0)
+})
+
+test('Test 1: propertly returns correct similarity method', () => {
```

Comment:
Same comments here, no need to index the test and can you add a bit more description to the test? This could be:

```suggestion
test('returns 1 for a mongod db index where the little endian is 1', () => {
```